src/main/java/leduc/command/EventCommand.java

```diff
@@ -69,7 +72,23 @@ else if(dateString[0].isBlank() || dateString[1].isBlank()){
             Date date1 = new Date(dateString[0]);
             Date date2 = new Date(dateString[1]);
             tasks.verifyConflictDate(date1, date2);
-            EventsTask newTask = new EventsTask(description, date1 , date2);
+            EventsTask newTask = null;
+            if (prioritySplit.length == 1){
+                newTask = new EventsTask(description, date1,date2);
+            }
+            else {
+                int priority = -1 ;
+                try{
+                    priority = Integer.parseInt(prioritySplit[1].trim());
+                }
+                catch(Exception e){
+                    throw new PrioritizeLimitException();
+                }
+                if (priority < 0 || priority > 9) {
+                    throw new PrioritizeLimitException();
+                }
+                newTask = new EventsTask(description,"[✗]",date1,date2,priority);
```

Comment:
why did you specify [✗] ?

Reply:
These constructors for different Tasks are already used in storage. It is to avoid to create an other constructor or to modify the first constructor and I will have to create a task by input the standard priority 5 in each instantiation in hard.

Do you think it is better to create another constructor or modify the first one ?